### PR TITLE
SQS Wildcard policy documentation

### PR DESCRIPTION
**Pull request by danielmarbach**

### CODE REVIEW

File: transports/sqs/configuration-ooptions-policy-intro.include.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+NServiceBus automatically subscribes to all event types an endpoint has handlers for. So given an endpoint with the following two handlers:
```
Comment (yvesgoeleven) [inline transports/sqs/configuration-ooptions-policy-intro.include.md:1]: The word 'So given' had me expecting a result directly after the listing. Given... then... 

Instead there is different text explaining autosubscribe.

Consider rewording?

File: transports/sqs/configuration-options_policy_sqs_[5,5.2).partial.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+#### Prefix condition
+
+Allow all messages from any topic with prefix to pass. No value needs to be provided, prefix from the [topic name prefix option](#topicnameprefix) will be used.
+
+TBD snippets
+
+#### Namespace condition
+
+Allow all messages from any message in the specified namespaces to pass.
```
Comment (yvesgoeleven) [inline transports/sqs/configuration-options_policy_sqs_[5,5.2).partial.md:44]: Namespace must be provided

Comment (danielmarbach) [reply transports/sqs/configuration-options_policy_sqs_[5,5.2).partial.md:44]: This is still WIP

File: transports/sqs/configuration-options_policy_sqs_[5.3,).partial.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+#### Prefix condition
+
+Allow all messages from any topic with prefix to pass. No value needs to be provided, prefix from the [topic name prefix option](#topicnameprefix) will be used.
+
+snippet: wildcard-prefix-condition
+
+#### Namespace condition
+
+Allow all messages from any message in the specified namespaces to pass.
```
Comment (lailabougria) [inline transports/sqs/configuration-options_policy_sqs_[5.3,).partial.md:46]:
```suggestion
Allow all messages in the specified namespaces to pass.
```

File: transports/sqs/configuration-options_policy_sqs_[5.3,).partial.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+#### Acount condition
+
+Allow all messages from any topic in the account to pass. No value needs to be provided, the account name will be derived from the subscribed topic ARN.
+
+snippet: wildcard-account-condition
+
+#### Prefix condition
+
+Allow all messages from any topic with prefix to pass. No value needs to be provided, prefix from the [topic name prefix option](#topicnameprefix) will be used.
```
Comment (lailabougria) [inline transports/sqs/configuration-options_policy_sqs_[5.3,).partial.md:40]:
```suggestion
Allow all messages from any topic with the prefix to pass. No value needs to be provided, the prefix from the [topic name prefix option](#topicnameprefix) will be used.
```

File: transports/sqs/configuration-options_policy_sqs_[5.3,).partial.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
+## Policy
+
+include: configuration-options-policy-intro
+
+The transport creates one policy statement with a combined condition for all events it subscribes to. An example policy for the above two subscribed events would look like the following:
```
Comment (lailabougria) [inline transports/sqs/configuration-options_policy_sqs_[5.3,).partial.md:5]:
```suggestion
The transport creates one policy statement with a combined condition for all events it subscribes to. An example policy for the events described above would look as follows:
```

File: transports/sqs/configuration-options_policy_sqs_[5,5.2).partial.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
+## Policy
+
+include: configuration-options-policy-intro
+
+The transport creates a policy statement with per events it subscribes to. An example policy for the above two subscribed events would look like the following:
```
Comment (lailabougria) [inline transports/sqs/configuration-options_policy_sqs_[5,5.2).partial.md:5]:
```suggestion
The transport creates a policy statement per event it subscribes to. An example policy for the event subscriptions described here above would look as follows:
```

**Pull request merged**